localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -92,6 +97,10 @@ class NDTScanMatcher : public rclcpp::Node
   void callback_regularization_pose(
     geometry_msgs::msg::PoseWithCovarianceStamped::ConstSharedPtr pose_conv_msg_ptr);
 
+  void set_initial_pose(
+    const geometry_msgs::msg::PoseWithCovarianceStamped::ConstSharedPtr initial_pose_msg_ptr);
+  bool process_scan_matching(
```

Comment:
If the function that performs the main processing of `service_ndt_align` is named `service_ndt_align_main`, should the function corresponding to `callback_initial_pose` be `callback_initial_pose_main`, and the function corresponding to `callback_sensor_points` should be `callback_sensor_points_main`?

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/5076/commits/fb5506937640da167b271a198a27b96f2477915f